### PR TITLE
Newsletters: add #37 (2019-03-12)

### PR DESCRIPTION
**Pull request by harding**
Todo:

- [x] Add image

(So sorry this is so late!)

### CODE REVIEW

File: _posts/en/newsletters/2019-03-12-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+  proposes to mitigate the problem of `OP_CODESEPARATOR` being used to
+  increase worse-case block verification time by instead increasing the
+  weight (vbytes) of transactions whose evaluated scripts contain the
+  opcode.  This would reduce the maximum number of code separators that
+  could be contained within a block while also likely reducing the
+  overall size and total number of operations in the block to the point
+  where it could be verified reasonably quickly.
+
+    O'Connor also raised a [similar concern][roconnor sighash] regarding
+    the soft fork's proposal to remove unallocated sighash type bytes.
```
Comment (moneyball) [inline _posts/en/newsletters/2019-03-12-newsletter.md]: Would it be better to say "treat unallocated sighash type bytes as invalid"? (vs. removing bytes)

**Comment by moneyball** [comment]
tACK. Other than my minor comment (which is optional to address) this newsletter looks ready to go! Oh, and the fixme image.

File: _posts/en/newsletters/2019-03-12-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+  whose nodes are based on the affected Bitcoin Core versions.  See the
+  disclosure details below for more information.
+
+- **Help test Bitcoin Core 0.18.0 RC1:** The first Release Candidate
+  (RC) for the next major version of Bitcoin Core has been released.
+  Organizations and experienced users who depend upon Bitcoin Core are
+  highly encouraged to [test it][0.18.0] for regressions and other
+  problems that could affect your use of it in production.  Any testing
+  is appreciated, but if you have some extra time after testing for your
+  specific use cases, please consider helping test the GUI.  This
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2019-03-12-newsletter.md]: This was a great idea, thanks!

**Comment by harding** [comment]
Addressed review comments (thanks!) and added deferred image.

**Comment by moneyball** [comment]
re-tested ACK

**Pull request merged**